examples/getting_started/2_Pipeline.ipynb

```diff
@@ -675,6 +674,34 @@
     "To support interactive zooming, where spreading would be needed only in sparse regions of the dataset, we provide the dynspread function.  `dynspread` will dynamically calculate the spreading size to use by counting the fraction of non-masked bins that have non-masked neighbors; see the\n",
     "[dynspread docs](https://datashader.org/api.html#datashader.transfer_functions.dynspread) for more details.\n",
     "\n",
+    "It is worth noting that older versions of datashader only supported the\n",
+    "use of spread and dynspread on images containing RGBA data. This\n",
+    "approach is still supported:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tf.spread(img, mask=mask)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Although this approach continues to works and can be more concise than\n",
+    "aggregate spreading, it is *not* recommended as the operation is less\n",
```

Comment:
```suggestion
    "Although both types of spreading are supported, RGB spreading\n",
    "is *not* recommended as the operation is less\n",
```

I don't think either approach has an overall advantage for concise code, because there are cases where agg spreading is much more concise, e.g. if you spread the agg once and then colormap it multiple times.  In that particular example we're showing multiple spreading types, but in general people probably look more often at different colormaps than at different spreading shapes, so I'd suspect that agg-based spreading is actually shorter overall. In any case it's not really relevant to mention here.

Reply:
Sure. I guess what I wrote was based on the particular example I was dealing with and probably can't be generalized to how everyone might be using datashader.